### ISSUE TITLE
Axs15231 transform issues

### ISSUE THREAD

**Issue by Hannah-GBS** [buglloc/esphome-components#17]
I'm having the issue where using `rotation: 270` causes the display updates to be very slow if a lot of pixels are changing, and I read that `rotation` is software based, whereas `transform` is hardware based, but it does not appear to work correctly with the axs15231 component at the moment. Setting any of the `transform` properties (`swap_xy`, `mirror_x`, and `mirror_y`) to True (and commenting out `rotation`), causes the T-Display Long to show a very glitchy version of my LVGL page on part of the screen, then it repeats this ~3 times. I've tried switching the `height` and `width` properties and all combinations of the 3 `transform` properties to no success.

![image](https://github.com/user-attachments/assets/00d308d1-e89a-4d9f-93c0-d44e1ae1c422)

I'm not sure if this issue is present in the dev version of the `qspi_dbi` core component this is based on, or what ETA there is for that updated component to be released. I'll probably try the dev build tomorrow but I don't know to what extent it supports this display either.

**Comment by buglloc** [comment]
> Setting any of the transform properties (swap_xy, mirror_x, and mirror_y) to True (and commenting out rotation), causes the T-Display Long to show a very glitchy version of my LVGL page on part of the screen

Yep, you are right. I didn't find any working version of hardware-based transformations yet, including official examples. I'll try to find a solution and most likely contact support next week

**Comment by tobiasrachl** [comment]
Did you find a solution to this issue? Would love to get the touchscreen running correctly but I don't have a clue on how to solve this issue...
